tests/integration/Mailer/WordPress/WordpressMailerTest.php

```diff
@@ -2,10 +2,14 @@
 
 namespace MailPoet\Mailer\WordPress;
 
+use MailPoet\DI\ContainerWrapper;
 use MailPoet\Entities\SubscriberEntity;
 use MailPoet\Mailer\Mailer;
 use MailPoet\Mailer\MailerError;
 use MailPoet\Mailer\MetaInfo;
+use MailPoet\Models\Subscriber;
```

Comment:
The new classes that are added here and are not related to WordPressMailer are just temporary and used in a test that needs to be refactored (or more likely moved somewhere else).